frontend/docker-setup.sh

```diff
@@ -8,11 +8,11 @@ if [ ! -f "$ENV_FILE" ]; then
     cp $BASEDIR/public/environment.docker.dist "$ENV_FILE"
 fi
 
-npm ci --verbose
-
 if [ "$CI" = 'true' ] ; then
+  npm ci --verbose
   npm run build
   npm run preview
 else
+  npm install
```

Comment:
We used npm ci here because this respects the lock file.
In our reviews, we have to check now if there are accidental changes to the package-lock.json.

Reply:
## Core Meeting Decision

If this happens, we can implement a check for the package.json and package-lock.json.
Also maybe we should use yarn instead of npm.